posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<p>Maybe you could expand this to talk about overheads? For example, if you would use again this same Numba function would it be faster as it is already compiled?, what would happen with numpy? etc Actually it might be nice to introduce "timings" even before this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>+1. Could discuss human overhead as well and link it to the precompiled code option. There is a real 'swings and roundabouts' type discussion that could be had here. These methods are great, but there is a time and place to implement them. Lists are fine for a quick and dirty notebook you're only going to run once, because the best you can do is save 6.6 milliseconds. On the other hand if you're looking to write some code that's going to be in production years from now, its worth spending hours getting that down to 0.1 milliseconds if that particular code block is going to be run billions of times.</p>